test/exe/krane_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'krane/cli'
+
+class KraneTest < KubernetesDeploy::TestCase
+  def test_version_prints_current_version
+    assert(krane.version)
+    assert_logs_match_all([
+      "Krane Version: #{KubernetesDeploy::VERSION}",
+    ], in_order: true)
+  end
+
+  private
+
+  def krane
+    @krane = Krane::CLI.new.tap do |krane|
+      krane.instance_variable_set('@logger', @logger)
```

Comment:
Hmmm, why do we need to do it like this? Won't it be instantiated transparently on the call to `#logger`?